### PR TITLE
Create sphinx_status

### PR DESCRIPTION
**Pull request by karelbarel**
simple pure bash plugin to display Sphinx status

### CODE REVIEW

File: plugins/sphinx/sphinx_status [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  [sphinx_*]
+  env.category mysql
+
+=head1 AUTHOR
+
+martin fuxa <root@unihost.cz>
+
+=head1 LICENSE
+
+CC
```
Comment (ssm) [inline plugins/sphinx/sphinx_status]: This (CC) looks like a reference to "Creative Commons", where there are multiple licenses, none if which are abbreviated "CC" . Could you specify or clarify the license?

**Comment by sumpfralle** [comment]
@karelbarel: are you still interested in this plugin? What do you think about the licence?

**Comment by sumpfralle** [comment]
@karelbarel: what do you think?

**Comment by sumpfralle** [comment]
please reopen (or comment), if you are still interested in submitting this plugin

**Pull request closed without merging**